product_docs/docs/pgd_cli/1/installing_cli.mdx

```diff
@@ -6,7 +6,7 @@ navTitle: "Installing the PGD CLI"
 
 ## Installing with TPAexec
 
-If you are using TPAexec to deploy EDB Postgres Distributed and the default deployment installs the PGD CLI package (`edb-pgd-cli`) on each BDR node automatically. It also creates the configuration file (`/etc/edb/pgd-config.yml`) prepopulated with the endpoints of each node in the cluster so no additional configuration is necessary.
+If you are using TPAexec to deploy EDB Postgres Distributed, it installs the PGD CLI package (`edb-pgd-cli`) on each BDR node, by default. It also creates the configuration file (`/etc/edb/pgd-config.yml`) prepopulated with the endpoints of each node in the cluster so no additional configuration is necessary.
```

Comment:
Sorry, forgot to mention earlier. Can we add TPAexec version (v22.14) which supports PGD CLI? Perhaps something like - 
```If you are using TPAexec (v22.14 and later) to deploy EDB Postgres Distributed... ```

TPAexec v22.14 is the latest version released last week along with PGD CLI which supports it.